pkg/api/addons_test.go

```diff
@@ -4311,11 +4311,7 @@ func TestSetAddonsConfig(t *testing.T) {
 					Containers: []KubernetesContainerSpec{
 						{
 							Name:  common.AzureNetworkPolicyAddonName,
-							Image: "mcr.microsoft.com/containernetworking/azure-npm:v1.0.29",
-						},
-						{
-							Name:  common.AzureVnetTelemetryContainerName,
-							Image: "mcr.microsoft.com/containernetworking/azure-vnet-telemetry:v1.0.29",
+							Image: "mcr.microsoft.com/containernetworking/azure-npm:v1.0.30",
```

Comment:
You will also have to update each UT case for which Azure NPM is enabled to expect the new `MemoryRequests` and `MemoryLimits` keys and values set as defaults.

Reply:
updated each UT w/ those